creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
-
-In Bedrock Edition, the `/schedule` command has the following syntax:
+In Bedrock Edition, its purpose is to run a function when an area of the world finishes loading:
 
 ```
-/schedule on_area_loaded add <from: x y z> <to: x y z> <function: filepath>
-/schedule on_area_loaded add circle <center: x y z> <radius: int> <function: filepath>
-/schedule on_area_loaded add tickingarea <name: string> <function: filepath>
+/schedule on_area_loaded add <from> <to> <function>
+/schedule on_area_loaded add circle <center> <radius> <function>
+/schedule on_area_loaded add tickingarea <name> <function>
 ```
 
-Rather than running a function after a certain period of time, functions can be scheduled to run when a certain region in the world is loaded. The "tickingarea" option will run the specified function when a ticking area of the specified name is loaded. If the ticking area is already active, then the function will run immediately. However, if the ticking area does not yet exist, the function will remain in limbo until the ticking area is created, such as with the `/tickingarea` command, after which the function will run.
+If the area is already active, the function will run immediately. Otherwise, it will wait until it becomes loaded. Unlike in Java Edition, scheduled functions cannot be cleared.
 
-Multiple functions can be scheduled for the same location or ticking area. Unlike Java Edition however, scheduled functions cannot be cleared.
+## Redstone
+Although redstone is typically superceded by commands and pack files, it's worth highlighting a few major differences.
 
-**Redstone** functions slightly different as well. Unlike Java Edition, Bedrock Edition doesn’t support quasi-connectivity. Systems that utilize mechanics such as Block Update Detector (BUD) switches won’t work. Pistons also require one tick to retract, and won’t leave blocks behind if given a one-tick pulse. Even the way updates happen is slightly different. While the vast majority of redstone circuits work well between the two versions, more complex circuits might not.
+[Quasi-connectivity](https://minecraft.fandom.com/wiki/Tutorials/Quasi-connectivity), a bug-turned-feature in Java Edition, is not present in Bedrock Edition. Systems that utilize mechanics such as "Block Update Detector" (BUD) switches won't work. Pistons also require one tick to retract, and won't leave blocks behind if given a one-tick pulse.
 
-**Resource Packs**
+While Java has a deterministic albeit directional order for redstone updates, Bedrock's update order is random, meaning a circuit that tries to have multiple blocks do something at the same moment may behave differently every time it's activated.
 
```

Comment:
I don't know much about redstone but I added the two differences I think are the most prominent